recipes/python-clingox/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = 'python-clingox' %}
+{% set version = '1.0.1' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/potassco/{{ name }}/archive/v{{ version }}.zip
+  sha256: c947596b38c2f08d39f433a974ea8a9661b9f342f171924eebf2b719dc1d9765
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - clingo
```

Comment:
No need for `clingo` in host requirements--only the things needed to run `setup.py` need to be listed.

Reply:
You are right. I was packaging C++ packages before and there it was necessary.